python/BioSimSpace/Process/_amber.py

```diff
@@ -895,7 +895,7 @@ def getElectrostaticEnergy(self, time_series=False, block="AUTO"):
         energy : :class:`Energy <BioSimSpace.Types.Energy>`
            The electrostatic energy.
         """
-        return self.getRecord("EELECT", time_series, _Units.Energy.kcal_per_mol, block)
```

Comment:
Why is this change?

Reply:
Because the key is wrong. All output that I generate for any protocol has `EELEC`, not `EELECT`. The commit message explains the reason for the change.